Berksfile

```diff
@@ -3,10 +3,8 @@ source 'https://supermarket.chef.io'
 metadata
 
 group :integration do
-  cookbook 'apt', '~> 3.0.0'
-  cookbook 'yum', '~> 3.10.0'
-  cookbook 'zypper', '~> 0.2.1'
-  cookbook 'pacman', '~> 1.1.1'
```

Comment:
was the removal of `zypper` and `pacman` intentional? Those were needed for arch and opensuse.